File: aspnet-core/src/Event_Planning_System.Core/Entities/SupplierPlaces.cs

using Abp.Domain.Entities;
using Event_Planning_System.Authorization.Users;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Event_Planning_System.Entities
{
	public class SupplierPlaces : Entity<int>
	{
		public EventCategory eventCategory { get; set; }
		public decimal Price { get; set; }//
		public string Name { get; set; }//
		public string Location { get; set; }
		public int Capacity { get; set; }//
		public string Image { get; set; }//
		public string ContactEmail { get; set; }//
		public string Description { get; set; }
		public long UserId { get; set; }
		[ForeignKey("UserId")]
		public virtual User User { get; set; }
	}
}

File: angular/src/shared/Models/SupplierPlaces.ts
export class SupplierPlaces {
        constructor(
                public id: number=0,
                public name: string | undefined='',
                public eventCategory: string | undefined='',
                public contactEmail: string | undefined='',
                public capacity: number | undefined=0,
                public price: number | undefined=0,
                public location: string | undefined='',
                public image: string | undefined='',
                public description: string | undefined='',
                public imagePath: File | undefined=undefined,
        ) {
        }
}